scenes/player/weapon/Weapon.tscn

```diff
@@ -0,0 +1,25 @@
+[gd_scene load_steps=4 format=2]
```

Comment:
I would put the /weapons directory inside of scenes/ but outside of scenes/player/